Core/DistortionPointHandler.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+using StarlightRiver.Core;
+using StarlightRiver.Helpers;
+
+using System;
+using System.Collections.Generic;
+
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.Graphics.Effects;
+using Terraria.Graphics.Shaders;
+
+namespace StarlightRiver.Core
+{
+    public delegate float DistortionIntensityFunction(float currentIntensity, int ticksPassed);
+
+    public delegate float DistortionProgressFunction(float currentProgress, int ticksPassed);
+
+    public delegate bool DistortionActiveFunction(float currentProgress, float currentIntensity, int ticksPassed);
+
+    public class DistortionPoint
+    {
+        public Vector2 Position;
+
+        public bool Active = true;
```

Comment:
Naming convention violation: fields should be lowerCamelCase

Reply:
Fixed